src/librustdoc/clean/types.rs

```diff
@@ -770,10 +791,9 @@ impl Item {
                 };
                 out.push(&int_s);
             }
-            if out.is_empty() {
-                return Vec::new();
```

Comment:
The early return wasn't correct. It resulted in us not rendering some attributes like `#[non_exhaustive]` in cross-crate re-export scenarios. I've added a regression test for this: `tests/rustdoc/inline_cross/attributes.rs`.